partner-onboarder/install.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# Onboards default partners 
+## Usage: ./install.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+echo "Do you have public domain & valid SSL? (Y/n) "
+echo "Y: if you have public domain & valid ssl certificate"
+echo "n: if you don't have public domain & valid ssl certificate"
```

Comment:
Update the above echo message

echo "n: If you don't have a public domain and a valid SSL certificate. Note: It is recommended to use this option only in development environments."